src/solvers/dgsem/basis_lobatto_legendre.jl

```diff
@@ -564,14 +559,13 @@ This implements algorithm 25 "GaussLobattoNodesAndWeights" from the book
   [DOI:10.1007/978-90-481-2261-5](https://doi.org/10.1007/978-90-481-2261-5)
 """
 # From FLUXO (but really from blue book by Kopriva)
```

Comment:
```suggestion
```
This comment separates the docstring from the function so that it does not appear in the public API.